publishable/database/migrations/2016_01_01_000000_add_user_avatar_and_role_id.php

```diff
@@ -6,25 +6,23 @@ class AddUserAvatar extends Migration
 {
     /**
```

Comment:
actually as those 2 fields are mandatory for the package to work correctly, maybe we can just call it `add_user_extra_fields.php` and ignore the drop part.